cloudbuild/configs.juniper

```diff
@@ -0,0 +1,124 @@
+VERSION=2.0.0
```

Comment:
Since we are only copying these configs, I think it's better to just leave the values that we want to override. Sultan will reads the values from `.configs` first and override them with the values from this file. This is good in case we release another version of the configs, and allows us a faster debug on cloud build.

It's a misassumption that we need to copy all the values to the new config file, I'll try fixing that later.

Reply:
I experimented with that approach and ran into problems because various variables depend on other ones. Sultan reads `.configs` first, interpolating any variables it sees there, then does the same with `.configs.cloudbuild`.

Eg, when it reads `.configs`, it sets `INSTANCE_NAME` to the result of `$USER-at-${HOST_NAME//./-}`, then sets `DENY_FIREWALL` and `ALLOW_FIREWALL` to `deny-$INSTANCE_NAME-webserver-access` and `allow-$INSTANCE_NAME-webserver-access`. Using that original value of `INSTANCE_NAME`. If we override `INSTANCE_NAME` in `.configs.cloudbuild` but don't also override `ALLOW_FIREWALL` and `DENY_FIREWALL`, they still have that original value with the `$USER-at-${HOST_NAME//./-}` stuff in it.

That becomes really confusing for maintainence since the person creating a new config needs to understand the entire dependency graph for the variables to know which ones need to get copied into the new config to override. It's much simpler to just copy `.configs` in its entirety.